packages/components/src/internal/components/domainproperties/propertiesUtil.ts

```diff
@@ -96,3 +97,112 @@ export function isFieldDeletable(field: DomainField): boolean {
 export function getVisibleFieldCount(domain: DomainDesign): number {
     return domain.fields.filter((field: DomainField) => field.visible).size;
 }
+
+export function compareStringsAlphabetically(a: string, b: string, direction): number {
+    const aStr = a ? a.toUpperCase() : '';
+    const bStr = b ? b.toUpperCase() : '';
+    const isAsc = direction === '+';
+
+    if (aStr < bStr) {
+        return isAsc ? -1 : 1;
+    }
+    if (aStr > bStr) {
+        return isAsc ? 1 : -1;
+    }
+    return 0;
+}
+
+export function removeFalseyObjKeys(obj) {
+    return Object.entries(obj).reduce((a, [k, v]) => (v ? ((a[k] = v), a) : a), {});
+}
+
+// columnOrder determines the left-to-right ordering of columns within the domain summary view
+export function reorderSummaryColumns(a: DomainPropertiesGridColumn, b: DomainPropertiesGridColumn): number {
+    const columnOrder = [
+        // Collapsed field options
+        'name',
+        'rangeURI',
+        'required',
+        'isPrimaryKey',
+        'lockType',
+        // Lookup options
+        'lookupContainer',
+        'lookupSchema',
+        'lookupQuery',
+        // Integer options
+        'format',
+        'defaultScale',
+        // Ontology options
+        'sourceOntology',
+        'conceptImportColumn',
+        'conceptLabelColumn',
+        'conceptURI',
+        // Expanded field options
+        'scale',
+        'description',
+        'label',
+        'importAliases',
+        'url',
+        'conditionalFormats',
+        'propertyValidators',
+        // Advanced Settings
+        'hidden',
+        'shownInUpdateView',
+        'shownInInsertView',
+        'shownInDetailsView',
+        'defaultValueType',
+        'defaultValue',
+        'defaultDisplayValue',
+        'phi',
+        'excludeFromShifting', // Appears for datetime fields
+        'measure',
+        'dimension',
+        'recommendedVariable',
+        'mvEnabled',
+    ];
+    return columnOrder.indexOf(a.index) > columnOrder.indexOf(b.index) ? 1 : -1;
+}
+
+export function removeUnusedProperties(obj) {
+    // only applicable in the QueryMetadata field editor case
+    delete obj.wrappedColumnName;
+    // Not surfaced in UI, and so removed from summary view
+    delete obj.propertyId;
+    delete obj.propertyURI;
+
+    return obj;
+}
+
+export function removeUnusedOntologyProperties(obj) {
+    if (!hasActiveModule('Ontology')) {
+        delete obj.sourceOntology;
+        delete obj.conceptImportColumn;
+        delete obj.conceptLabelColumn;
+        delete obj.conceptURI;
```

Comment:
this `conceptURI` isn't strictly for ontology, it should be added back and not grouped with these other 3 ontology properties.

Reply:
Just checking, is the only other case when the `Data Type` is `Sample`? I exercised some options and poked around in the code and am not seeing others. I've removed the line from `removeUnusedOntologyProperties()` and currently have it under `// Sample or Ontology option` in reorderSummaryColumns()

Reply:
The conceptURI is used for a few data types: ParticipantId, Flag, Sample, Ontology Lookup

Reply:
Gotcha! I've moved it to 'Other expanded field options' but flagged the line with 'ParticipantId, Flag, Sample, and Ontology Lookup' just to keep track of that info